src/sap/fhir/model/r4/FHIRListBinding.js

```diff
@@ -547,7 +562,7 @@ sap.ui.define([
 					 * @since 1.0.0
 					 */
 	FHIRListBinding.prototype._resetData = function() {
-		this.aKeys = undefined;
+		this.aKeys = [];
```

Comment:
Good point. Why we had reseted the `aKeys` to `undefined`?  If you change this here. You also have to chance the coding in the `getContexts` method where we initialize the `aKeys` and set the startIndex https://github.com/SAP/openui5-fhir/blob/530b31c9f1e1ebf2102774658a407793ee7b0c56/src/sap/fhir/model/r4/FHIRListBinding.js#L113

Reply:
@flovogt : I am not sure why this alone was undefined.. However i had changed the get contexts to not do the undefined check and set the start index directly from the array length